ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -16,12 +25,127 @@ use std::ffi::{CStr, CString};
 use std::mem;
 use std::os::raw::{c_char, c_void};
 use std::panic::{self, AssertUnwindSafe, UnwindSafe};
+use std::thread;
```

Comment:
This will need either a #[cfg] for windows-only, or to be moved into the function that uses it, otherwise we'll get unused import warnings.

Reply:
Yes, good point.  I've moved it with the rest of the "use …" statements and placed in the function scope as follows:
fn do_redirect_stdout_stderr() -> Result<(), ()> {
    **use std::thread;**
    use winapi::shared;
    use winapi::um::debugapi;
    use winapi::um::handleapi;
    use winapi::um::minwinbase;
    use winapi::um::namedpipeapi;
    use winapi::um::processenv;
    use winapi::um::winbase;
    use winapi::um::winnt;